samples/cellular/modem_shell/README.rst

```diff
@@ -1182,79 +1160,48 @@ Application FOTA support
 To build the MoSh sample with application FOTA support, use the ``-DOVERLAY_CONFIG=overlay-app_fota.conf`` option.
 For example:
 
-.. tabs::
-
-   .. group-tab:: nRF9161 DK
+.. parsed-literal::
+   :class: highlight
 
-      .. code-block:: console
+   west build -p -b *build_target* -d build -- -DOVERLAY_CONFIG=overlay-app_fota.conf
 
-         west build -p -b nrf9161dk_nrf9161_ns -d build -- -DOVERLAY_CONFIG=overlay-app_fota.conf
-
-   .. group-tab:: nRF9160 DK
-
-      .. code-block:: console
-
-         west build -p -b nrf9160dk_nrf9160_ns -d build -- -DOVERLAY_CONFIG=overlay-app_fota.conf
+|build_target|
 
 nRF91 Series DK with full modem FOTA support
 ============================================
 
 To build the MoSh sample for an nRF91 Series DK with full modem FOTA support, use the devicetree overlay for external flash corresponding to your device and the ``-DOVERLAY_CONFIG=overlay-modem_fota_full.conf`` option.
-For example:
+The following is an example for the nRF9161 DK:
 
-.. tabs::
-
-   .. group-tab:: nRF9161 DK
-
-      .. code-block:: console
-
-         west build -p -b nrf9161dk_nrf9161_ns -d build -- -DOVERLAY_CONFIG=overlay-modem_fota_full.conf -DDTC_OVERLAY_FILE=nrf9161dk_ext_flash.overlay
-
-   .. group-tab:: nRF9160 DK
-
-      .. code-block:: console
+.. code-block:: console
 
-         west build -p -b nrf9160dk_nrf9160_ns -d build -- -DOVERLAY_CONFIG=overlay-modem_fota_full.conf -DDTC_OVERLAY_FILE=nrf9160dk_ext_flash.overlay
+   west build -p -b nrf9161dk_nrf9161_ns -d build -- -DOVERLAY_CONFIG=overlay-modem_fota_full.conf -DDTC_OVERLAY_FILE=nrf9161dk_ext_flash.overlay
```

Comment:
Do we want to remove the example on how it's compiled for 9160dk?

Reply:
I was thinking on why I was bothered about not having 9160 compilation command. I think the reason is that there are few configurations (full app fota, ESP8266, bt shell) where we give an example compilation command for a specific target. And for the rest we have the generic *build_target*.
Now I realize that I'm thinking what are the DKs where these full app fota and ESP8266 are supported. For BT shell, there's a note about it.
We could have nrf9161dk replaced with *target_version* or whatever in` nrf9161dk_ext_flash.overlay` but that would require specifying the *target_version*.

I'm fine leaving the documentation as it is now in the PR. I'm supposed to discuss about a bigger rework for modem shell documentation anyway in the future.